script/pmltq

```diff
@@ -747,3 +861,143 @@ sub apply_filter {
   }
   $filter->{finish}->($filter) if $filter;
 }
+
+sub load_config {
+  my $config_file = shift;
+  my $data;
+  eval {$data = YAML::Tiny->read($config_file)};
+  if ($@ && $@ =~ m/YAML_LOAD_ERR/) {
+    print STDERR "unable to load config file '$config_file'\n";
+  } elsif ($@ && $@ =~ m/YAML_PARSE_ERR/) {
+    $@ =~ s/\n.*//g;
+    print STDERR "unable to parse config file '$config_file'\n\t$@\n";
+  } elsif (! $data) {
+    print STDERR "unable to open config file '$config_file'\n";
+  }
+  exit unless $data;
+  $data->[0]->{db}->{driver} ||= 'Pg';
+  return $data->[0];
+}
+
+sub db_connect {
+  my $config = shift;
+  my $dbname = shift || $config->{db}->{name};
+  die "Database driver ".$config->{db}->{driver}." is not supported !!!\n" unless $config->{db}->{driver} eq 'Pg';
+  my $dbh_ = DBI->connect("DBI:".$config->{db}->{driver}.":dbname=$dbname;host=".$config->{db}->{host}.";port=".$config->{db}->{port}, 
+    $config->{db}->{user}, 
+    $config->{db}->{password}, 
+    { RaiseError => 0, PrintError => 1, mysql_enable_utf8 => 1 }) or die "$error_msg{connection}\n$DBI::errstr";
+  return $dbh_;
+}
+
+sub db_disconnect {
+  my $dbh_ = shift;
+  $dbh_->disconnect();
+}
+
+sub cmd_convert {
+  my $config = load_config(shift);
+  my $sqldir = shift;
+  if(!-d $sqldir) {
+    make_path($sqldir) or die "Unable to create directory $sqldir\n";
+  }
+  Treex::PML::AddResourcePath($config->{resources});
+  for my $layer (@{$config->{layers}}) {
+    print STDERR "Converting data for layer $layer->{name}\n";
+    PMLTQ::PML2BASE::init();
```

Comment:
Calling init second time doesn't clear variables set from previous layer. That is causing errors - I think.